### PR TITLE
Adding support for outputting in Markdown format

### LINKED ISSUE #38: Request: Output as HTML or Markdown.
It would be very cool if there was an option to output to HTML or Markdown.

Reason:
Then we can automaticly put it on a storageblob with static webpage.
this way we have an updated documentation webpage.

### PR DESCRIPTION
**Pull request by jacobcsmith**
I tried to minimize changes to the existing code by creating proxy functions for writing content to the documentation. The extension of the FullDocumentationPath parameter will determine if Word or Markdown format is used. 

Closes #38.

### CODE REVIEW

File: PSModule/IntuneDocumentation/Functions/Invoke-IntuneDocumentation.ps1 [hunk 1]
```diff
@@ -128,35 +129,37 @@ Function Invoke-IntuneDocumentation(){
             $Intune_App | Add-Member Noteproperty "DisplayName" $_.displayName
             $Intune_App | Add-Member Noteproperty "Type" (Format-MsGraphData $_.'@odata.type')
             $Assignments = @()
+            $newline = if ($format -eq 'md') { '<br>' } else { "`n" }
```
Comment (jacobcsmith) [inline PSModule/IntuneDocumentation/Functions/Invoke-IntuneDocumentation.ps1:132]: A newline was breaking the Markdown table

**Comment by ThomasKur** [comment]
Cool to see this extension. I will validate it on the weekend and then merge it

**Comment by Blankf** [comment]
Any update @ThomasKur on your view?

**Comment by ThomasKur** [comment]
Sorry for the long delay. Please keep in mind that I currently just manage the new project https://github.com/ThomasKur/M365Documentation

Regards
Thomas

**Pull request merged**